artemis/src/test/java/tech/pegasys/artemis/cli/BeaconNodeCommandTest.java

```diff
@@ -174,7 +165,7 @@ public void overrideDefaultValuesIfKeyIsPresentInConfigFile() throws IOException
   @Test
   public void interopEnabled_shouldNotRequireAValue() {
     final ArtemisConfiguration artemisConfiguration =
-        getArtemisConfigurationFromArguments(INTEROP_ENABLED_OPTION_NAME);
+        getArtemisConfigurationFromArguments("--Xinterop-enabled");
```

Comment:
I thought it was kind of a nice feature to not have to track down these strings when we change a CLI option name

Reply:
Yeah it does have some advantages, but changing CLI option names should be very rare and very deliberate as it breaks backwards compatibility for users.